.github/workflows/build_and_run_apps.yml

```diff
@@ -0,0 +1,122 @@
+name: Build and Run Apps
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Once per day at midnight
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  build:
+    name: Build Apps
+    strategy:
+      fail-fast: false
+      matrix:
+        idf_ver: 
+          - "release-v5.0"
+          - "release-v5.1"
+          - "release-v5.2"
+          - "release-v5.3"
+          - "latest"
+        parallel_index: [1,2,3,4,5] # This must from 1 to 'parallel_count' defined in .idf_build_apps.toml
+    runs-on: ubuntu-22.04
+    container: espressif/idf:${{ matrix.idf_ver }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+      - name: Install dependencies
+        shell: bash
+        run: |
+          . ${IDF_PATH}/export.sh
+          pip install idf-component-manager idf-build-apps --upgrade
+      - name: Find Apps
+        shell: bash
+        run: |
+          . ${IDF_PATH}/export.sh
+          idf-build-apps find
+      - name: Build Apps
+        shell: bash
+        run: |
+          . ${IDF_PATH}/export.sh
+          export PEDANTIC_FLAGS="-DIDF_CI_BUILD -Werror -Werror=deprecated-declarations -Werror=unused-variable -Werror=unused-but-set-variable -Werror=unused-function"
+          export EXTRA_CFLAGS="${PEDANTIC_FLAGS} -Wstrict-prototypes"
+          export EXTRA_CXXFLAGS="${PEDANTIC_FLAGS}"
+          idf-build-apps build --parallel-index ${{ matrix.parallel_index }}
```

Comment:
```suggestion
          idf-build-apps build --parallel-index ${{ matrix.parallel_index }} --parallel-count 5
```

Reply:
It's set in idf_build_apps.toml, but sure I can move it here.

Reply:
Okay, I see, it's better to define this on the command line, otherwise it affects local execution of `idf-build-apps build` as well...